docs/src/main/sphinx/security/opa-access-control.md

```diff
@@ -52,6 +52,10 @@ The following table lists the configuration properties for the OPA access contro
 * - `opa.policy.column-masking-uri`
   - The **optional** URI for fetching column masks - if not set no masking
     is applied. For example, `https://opa.example.com/v1/data/columnMask`.
+* - `opa.policy.batch-column-masking-uri`
+  -  The **optional** URI for fetching columns masks in batches; should **not**
+     be used with `opa.policy.column-masking-uri`. For example,
+     `http://opa.example.com/v1/data/batchColumnMasks`.
```

Comment:
is this the default URI path from an OPA server?

Reply:
The is no default URI path; the path is determined by the Rego policy, roughly `/v1/data/{package.replace(".", "/")}/{rule}`.
For the example Rego this PR adds the path would be `/v1/data/trino/batchColumnMasking`

Reply:
ok.. still makes this a reasonable example then. Maybe its worth doing /v1/data/example/batchColumnMasking ?

Reply:
Far point, checked offline with @vagaerg and I think `/v1/data/batchColumnMasks` is not actually a valid path as it has no package (though there may be some OPA config which allows this that I am not aware of).
Updated all paths in the doc.